docs/modules/ROOT/pages/how-tos/synthesize.adoc

```diff
@@ -0,0 +1,55 @@
+= Synthesize a cluster
+
+include::partial$synthesize-macros.adoc[]
+
+TIP: "Synthesize" is the act of making a cluster managed by Project Syn
+
+This how-to will show you how to synthesize a cluster.
+
+== Prerequisites
+
+* `cluster-admin` permissions on the cluster to synthesize.
+* Write access in the Project Syn "Clusters" view on {portal-lieutenant-clusters}.
+* Access to the {synfra-cluster}.
+* The Syn tenant of the cluster to synthesize exists already.
+If the tenant doesn't exist yet, see the documentation on xref:how-tos/create-tenant.adoc[creating a tenant].
+
+== Procedure
+
+. Create a new cluster in https://control.vshn.net/syn/lieutenantclusters/vshn-lieutenant-prod/_create[control.vshn.net].
+See xref:explanation/synthesize.adoc#_creating_a_new_cluster_on_control_vshn_net[section "creating a new cluster on control.vshn.net"] of the "Synthesize a cluster" explanation for a more detailed breakdown of the form fields.
+
+. Run Commodore to create a local checkout of the new cluster's configuration.
+See the Project Syn https://syn.tools/syn/how-tos/compile-catalog.html["compile a catalog" how-to].
+Use `\https://api.syn.vshn.net` as the value for `LIEUTENANT_URL`.
+Skip the "Cleanup" of that tutorial step for now.
+
+. Add any required initial configurations to your new cluster in `inventory/classes/<tenant-id>/<cluster-id>.yml`
+
+. Run `commodore catalog compile --push <cluster-id>` to push the initial cluster catalog to https://git.vshn.net[git.vshn.net].
+
+. Create any required secrets in Vault.
+See the section describing how to <<_list_referenced_secrets,list referenced secrets>> to generate a list of all secrets referenced in the cluster catalog.
+
+. Fetch the install URL from the {synfra-cluster}.
```

Comment:
I'd replace this by the Portal integration. It's getting fixed in PORTAL-50 and can could currently be work-arounded by simply refreshing the page.

Reply:
added a note for now that a refresh is required, we can always change the text to refer to the new implementation once it's live.